tests/validator/test_validation_graph.py

```diff
@@ -38,14 +50,62 @@ def expect_column_values_to_be_unique_expectation_config() -> ExpectationConfigu
 
 
 @pytest.fixture
-def expectation_validation_graph(
+def expect_column_value_z_scores_to_be_less_than_expectation_config() -> ExpectationConfiguration:
+    return ExpectationConfiguration(
+        expectation_type="expect_column_value_z_scores_to_be_less_than",
+        kwargs={
+            "column": "a",
+            "mostly": 0.9,
+            "threshold": 4,
+            "double_sided": True,
+        },
+    )
+
+
+@pytest.fixture
+def expect_column_values_to_be_unique_expectation_validation_graph(
     expect_column_values_to_be_unique_expectation_config: ExpectationConfiguration,
 ) -> ExpectationValidationGraph:
     return ExpectationValidationGraph(
         configuration=expect_column_values_to_be_unique_expectation_config
     )
 
 
+@pytest.fixture
+def expect_column_value_z_scores_to_be_less_than_expectation_validation_graph(
+    expect_column_value_z_scores_to_be_less_than_expectation_config,
+):
+    class PandasExecutionEngineStub:
+        pass
+
+    PandasExecutionEngineStub.__name__ = "PandasExecutionEngine"
+    pandas_execution_engine_stub = cast(ExecutionEngine, PandasExecutionEngineStub())
+
+    expectation_configuration = ExpectationConfiguration(
+        expectation_type="expect_column_value_z_scores_to_be_less_than",
+        kwargs={
+            "column": "a",
+            "mostly": 0.9,
+            "threshold": 4,
+            "double_sided": True,
+        },
+    )
+    graph = ValidationGraph(execution_engine=pandas_execution_engine_stub)
+    validation_dependencies = (
+        ExpectColumnValueZScoresToBeLessThan().get_validation_dependencies(
+            expectation_configuration, pandas_execution_engine_stub
+        )
+    )
+
+    for metric_configuration in validation_dependencies["metrics"].values():
+        graph.build_metric_dependency_graph(
+            metric_configuration=metric_configuration,
+            runtime_configuration=None,
+        )
+
+    return graph
+
+
```

Comment:
I find this parameterization confusing. I would rather see this broken into two tests, one for table, one for column, and have a (non-test) method contain the assertions to reduce the duplication.

On deeper inspection, this is testing a simple assignment constructor. This test, `test_MetricEdge_init`, can be removed

Reply:
@donaldheppner I agree -- @cdkini do you mind if I delete this test, or if you created with a specific purpose in mind, please let us know soon? Thanks!

Reply:
@donaldheppner Deleted (in next commit) after confirming with @cdkini. Thanks.